src/Components/Layout/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,82 @@
+import { useState, FC, ReactNode, Key } from "react";
+import type { MenuProps } from "antd";
+import { Layout, Menu } from "antd";
+import "./Sidebar.css";
+import {
+  BorderlessTableOutlined,
+  HomeOutlined,
+  BellOutlined,
+  TeamOutlined,
+  UserOutlined,
+  MoreOutlined,
+  UserSwitchOutlined,
+  SettingOutlined,
+  BulbOutlined,
+  UserDeleteOutlined,
+} from "@ant-design/icons";
+import { useNavigate } from "react-router-dom";
+
+const { Sider } = Layout;
+type MenuItem = Required<MenuProps>["items"][number];
+
+export const Sidebar: FC = () => {
+  const [collapsed, setCollapsed] = useState(false);
+  const navigate = useNavigate();
+  const handleCollapse = (collapsed: boolean) => {
+    console.log(collapsed);
```

Comment:
Console can be removed..if not required..

Reply:
done